[docs] Update issues, questions and contributing (#3012)
DevExpress/testcafe: README.md

* [IDE for End-to-End Web Testing](#ide-for-end-to-end-web-testing)
* [Getting Started](#getting-started)
* [Documentation](#documentation)
* [Get Help](#get-help)
* [Stay in Touch](#stay-in-touch)

> * [Issue tracker](#issue-tracker)

Reviewer: ```suggestion
* [Issue Tracker](#issue-tracker)
```

Author: Done

## Documentation

Go to our website for full [documentation](https://devexpress.github.io/testcafe/documentation/getting-started/) on TestCafe.

## Get Help

Join the TestCafe community on Stack Overflow to get help. Ask and answer [questions with the TestCafe tag](https://stackoverflow.com/questions/tagged/testcafe).

> ## Stay in Touch

Author: Good idea, done.

> Go to our website for full [documentation](http://devexpress.github.io/testcafe/documentation/using-testcafe/) on TestCafe.

Reviewer: ```suggestion
Go to our website for full [documentation](https://devexpress.github.io/testcafe/documentation/getting-started/) on TestCafe.
```

## Get Help

Join the TestCafe community on Stack Overflow to get help. Ask and answer [questions with the TestCafe tag](https://stackoverflow.com/questions/tagged/testcafe).

## Stay in Touch

Follow us on [Twitter](https://twitter.com/DXTestCafe). We post TestCafe news and updates, several times a week.

## Issue Tracker

Use our GitHub issues page to [report bugs](https://github.com/DevExpress/testcafe/issues/new?template=bug-report.md) and [suggest improvements](https://github.com/DevExpress/testcafe/issues/new?template=feature_request.md).

## Contributing

Read our [Contributing Guide](https://github.com/DevExpress/testcafe/blob/master/CONTRIBUTING.md) to learn how to contribute to the project.

> To create your own plugin for TestCafe you can use these plugin generators:

Reviewer: ```suggestion
To create your own plugin for TestCafe, you can use these plugin generators:
```

## Community

> Join the TestCafe community on Stack Overflow to get help. Ask and answer [questions with the TestCafe tag](https://stackoverflow.com/questions/tagged/testcafe).

Reviewer 2: So, we expect people to look for info in the `Community` section in case they need help? I wouldn't think that I need `Community` in this case.

What about a separate section `Get Help`?

Author: I had similar doubts. You confirmed them, will change.

## Community

Join the TestCafe community on Stack Overflow to get help. Ask and answer [questions with the TestCafe tag](https://stackoverflow.com/questions/tagged/testcafe).

Follow us on [Twitter](https://twitter.com/DXTestCafe) to stay in touch. We post TestCafe news and updates, several times a week.

## Issue Tracker

Use our GitHub issues page to [report bugs](https://github.com/DevExpress/testcafe/issues/new?template=bug-report.md) and [suggest improvements](https://github.com/DevExpress/testcafe/issues/new?template=feature_request.md).

> 

Reviewer 3: Remove this experience empty line, that's why tests fail
